### PR TITLE
Add org.kde.kwrite (Kwrite)

### PR DESCRIPTION
**Pull request by Siosm**
This is a special case as this application is probably already provided by org.kde.kate (Kate). Can we provide several application with the same flatpak or do we have to make one for each like this one? The manifest is almost the same.

### CODE REVIEW

File: org.kde.kwrite.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "kwrite",
+    "rename-icon": "kwrite",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=cups",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
```
Comment (bilelmoussaoui) [inline org.kde.kwrite.json:13]:
```suggestion
        "--device=dri"
```

**Comment by nedrichards** [comment]
You can export multiple desktop files from the same flatpak (e.g. LibreOffice does this to offer different entry points for the different apps). It's a bit of a fiddle though. You need to export them both with the same prefix so in this case it'd be `org.kde.kate` and `org.kde.kate-kwrite` I think.

Certainly in this case it'd be interesting to see how much deduplication you get from ostree "for free" (e.g. what the client side overhead of two apps would be - I'm aware there's also a developer side cost to keeping two things in sync.)

**Comment by Siosm** [comment]
@flathub/kde

**Comment by Siosm** [comment]
I took a look at the LibreOffice manifest but could not find anything specific regarding this. I'm interested as I have two other applications that would benefit from it: [Massif Visualizer & KGraphViewer](https://github.com/flathub/flathub/pull/1581).

**Comment by nedrichards** [comment]
bot, build org.kde.kwrite

**Comment by nedrichards** [comment]
@Siosm here's an old issue talking about it. https://github.com/flathub/org.libreoffice.LibreOffice/issues/36 I believe the code that does this for them is all upstream now. You can also ask about this on #flatpak IRC and you might find some helpful info.

**Comment by Siosm** [comment]
bot, build org.kde.kwrite

**Comment by Siosm** [comment]
I have updated this one which should be ready.

Thinking about the Kate merge, I think it would better to have both Kate & kwrite as distinct apps in FlatHub to make them more discoverable. Maybe I should make them depend on a common module with a git submodule?

@tsdgeos What do you think? What would you prefer here?

**Comment by tsdgeos** [comment]
I guess it does indeed increase discoverability and the filesize of this thing seems to be like 8MB so that's not a problem either.

One thing before we mege, remove the konsole dependency, kwrite doesn't show the plugins like kate does, so it's impossible to open the konsole plugin from kwrite so we don't need to compile it :)

**Comment by Siosm** [comment]
bot, build org.kde.kwrite

**Comment by tsdgeos** [comment]
Looks good to me.

@bilelmoussaoui unless you have some other things to improve, please merge :)

**Comment by Siosm** [comment]
bot, build org.kde.kwrite

**Comment by Siosm** [comment]
I added a commit to try removing the duplicated content with Kate.

**Comment by tsdgeos** [comment]
Still looks good

**Pull request closed without merging**